app/channels.go

```diff
@@ -31,6 +31,11 @@ type configSvc interface {
 	SaveConfig(newCfg *model.Config, sendConfigChangeClusterMessage bool) (*model.Config, *model.Config, *model.AppError)
 }
 
+type licenseSvc interface { // nolint: unused,deadcode
```

Comment:
Let's add a quick comment that this interface is only kept as a starting point for other products to use, and is not actually used in the code.